internal/pkg/skuba/upgrade/node/versions.go

```diff
@@ -95,6 +97,16 @@ func (nviu NodeVersionInfoUpdate) IsFirstControlPlaneNodeToBeUpgraded(client cli
 // If all preconditions are met for the given node, no error will be returned.
 func (nviu NodeVersionInfoUpdate) NodeUpgradeableCheck(client clientset.Interface, currentClusterVersion *version.Version) error {
 	errorMessages := []string{}
+	if strings.HasPrefix(nviu.Update.KubeletVersion.String(), skubaconstants.LastCaaSP4KubernetesVersion) {
+		errorMessages = append(errorMessages, fmt.Sprintf("Cannot use this version of skuba to upgrade to kubernetes version %s. Please use a skuba version below 2.0.0 instead.", skubaconstants.LastCaaSP4KubernetesVersion))
```

Comment:
this version is a bit confusing too me. skuba version 2.0.0? should this not be caasp versions? to be honest. I don't even know what `skuba version 2.0.0` is.

Reply:
Can you propose an alternative? I used SP1 before and it was confusing.

Reply:
Would "skuba from CaaSP 4 on SLES 15 SP1 (version below 2.0)" be better?

I will do that in a separate PR.

Reply:
I like that, thought I am still not sure about adding the `version 2.0` stuff as unless you look under `tags` of this repository it is meaningless. even our branching is based off of CaaSP version. but that is fine.